pkg/spec/headers.go

```diff
@@ -21,19 +21,29 @@ import (
 	"github.com/go-openapi/spec"
 )
 
-var ignoredHeaders = map[string]bool{
-	contentTypeHeaderName:       true,
-	acceptTypeHeaderName:        true,
-	authorizationTypeHeaderName: true,
+var defaultIgnoredHeaders = []string{
+	contentTypeHeaderName,
+	acceptTypeHeaderName,
+	authorizationTypeHeaderName,
 }
 
-// TODO: Do we want to support adding ignored headers via an env var?
-func shouldIgnoreHeader(headerKey string) bool {
-	return ignoredHeaders[strings.ToLower(headerKey)]
+func createHeadersToIgnore(headers []string) map[string]struct{} {
+	ret := make(map[string]struct{})
+
+	for _, header := range append(defaultIgnoredHeaders, headers...) {
```

Comment:
dont you think it is better to do the append in a separate command?
c := append(defaultIgnoredHeaders, headers...)
for _, header := range c {
}

Reply:
why should I create a separate parameters for it?